apps/web/src/utils/experienceUtils.tsx

```diff
@@ -222,6 +232,68 @@ export const getExperienceFormLabels = (
       description:
         "Label displayed on experience form/card for how a skill was applied section",
     }),
+    classificationGroup: intl.formatMessage({
+      defaultMessage: "Group",
+      id: "kUqaoo",
+      description:
+        "Label displayed on Work Experience form for classification group input",
+    }),
+    classificationLevel: intl.formatMessage({
+      defaultMessage: "Level",
+      id: "Y7Qop6",
+      description:
+        "Label displayed on Work Experience form for classification level input",
+    }),
+    extSizeOfOrganization: intl.formatMessage({
+      defaultMessage: "Size of the organization",
+      id: "HP5PEg",
+      description: "Label for the size of the organization radio group",
+    }),
+    extRoleSeniority: intl.formatMessage({
+      defaultMessage: "Seniority of the role",
+      id: "34NvoS",
+      description: "Label for the seniority of the role radio group",
+    }),
+    govEmploymentType: intl.formatMessage({
+      defaultMessage: "Employment type",
+      id: "uaEMMO",
+      description: "Label for the employment type radio group",
+    }),
+    classification: intl.formatMessage({
+      defaultMessage: "Classification",
+      id: "d1FYv4",
+      description: "Label displayed on Work Experience card for classification",
+    }),
+    positionType: intl.formatMessage({
+      defaultMessage: "Position type",
+      id: "0Dp1N4",
+      description: "Label for the position type radio group",
+    }),
+    govContractorRoleSeniority: intl.formatMessage({
+      defaultMessage: "Seniority of the role",
+      id: "WsG9lo",
+      description: "Label for the role seniority radio group",
+    }),
```

Comment:
This looks like a duplication of line 252. Are these contextually different for French? :thinking: